pandapower/results_branch.py

```diff
@@ -680,6 +680,20 @@ def _get_switch_results(net, i_ft, suffix=None):
         with np.errstate(invalid='ignore'):
             i_ka = np.max(i_ft[f:t], axis=1)
         net[res_switch_df].loc[net._impedance_bb_switches, "i_ka"] = i_ka
+
+        pf_mw = ppc["branch"][f:t, PF].real
```

Comment:
I don't see a reason to have pf_mw and pt_mw if only p_from_mw and p_to_mw are used